opsmngr/system_measurements.go

```diff
@@ -0,0 +1,54 @@
+package opsmngr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+)
+
+const (
+	systemMeasurements = "groups/%s/hosts/%s/measurements"
+)
+
+// SystemMeasurementsService is an interface for interfacing with the System Measurements
+// endpoints of the MongoDB Ops Manager API.
+type SystemMeasurementsService interface {
+	List(context.Context, string, string, *atlas.ProcessMeasurementListOptions) (*atlas.ProcessMeasurements, *atlas.Response, error)
+}
+
+// ServiceMeasurementsServiceOp handles communication with the system measurements related methods of the
+// MongoDB Ops Manager API
+type ServiceMeasurementsServiceOp struct {
+	Client atlas.RequestDoer
+}
+
+var _ SystemMeasurementsService = &ServiceMeasurementsServiceOp{}
+
+// List lists system and process measurements on the CPU usage of the hosts that run MongoDB.
+// See more: https://docs.opsmanager.mongodb.com/current/reference/api/measures/get-host-process-system-measurements/
+func (s *ServiceMeasurementsServiceOp) List(ctx context.Context, projectID, host string, listOptions *atlas.ProcessMeasurementListOptions) (*atlas.ProcessMeasurements, *atlas.Response, error) {
```

Comment:
small nit, can we call host -> hostID, I know is confusing as atlas uses `host:port` but OM/CM use `hostID` 🤷‍♂